python/sparkdl/transformers/keras_applications.py

```diff
@@ -169,7 +176,7 @@ def _testKerasModel(self, include_top):
 
 class XceptionModel(KerasApplicationModel):
     def preprocess(self, inputImage):
-        return xception.preprocess_input(inputImage)
+        return xception.preprocess_input(_rgb2bgr(inputImage))
```

Comment:
One issue I have with the current proposal is that it the preprocess has different inputs and outputs, which makes it hard to track what is going on for the user. I would rather have `BGR` -> `BGR` or `RGB` -> `RGB` instead of a mix. Otherwise, users are going to be confused.

Reply:
Hm I think I don't understand what you mean here. 

All our interfaces should now take image schema's BGR order. I flip it to RGB here because the internal keras function expects it in that order.